README.md

```diff
@@ -1,47 +1,26 @@
 
-Trino
+Trino Community Kubernetes Helm Charts
 ===========
+[![Release Trino Chart](https://github.com/trinodb/charts/actions/workflows/release.yaml/badge.svg?branch=main)](https://github.com/trinodb/charts/actions/workflows/release.yaml)
 
 Fast distributed SQL query engine for big data analytics
```

Comment:
The text for the Trino GitHub changed to "Fast distributed SQL query engine for big data analytics that helps you explore your data universe" might want to update.